### ISSUE TITLE
Production tab in v1.x

### ISSUE THREAD

**Issue by brandonmcclure** [fluffy-mods/ColonyManager#157]
**Problem description**
This mod makes rimworld so much more fun to play! I have used it on older versions, and only recently started playing again with v1.x. I know that there were a lot of breaking changes in general between beta and release and I see on the readme file as well as the mod description in steam that the production tab is not enabled at the moment. 

**Describe the solution**
I can use ColonyManager to setup jobs that will control the production of bills across all of my production workbenches playing v1.x of Rimworld.

**Alternatives**
I have not found other mods that add the functionality I want. I recall it existing in the beta versions. 

**Additional materials**
I would like to assist if I can with this. I am unfamiliar with rimworld mod development, but know some c# and how to google/get going with a PR. Let me know where you would most like me to direct my attention.

**Comment by kbatbouta** [comment]
@FluffierThanThou May i ask why did you remove the production tab? I've been thinking that i would give it a try. However i can't find any reference to why it was discontinued even in the [original commit: code style cleanup ](https://github.com/fluffy-mods/ColonyManager/commit/06bf6915ca3d6a97e324dbd60ec317f9fcd080ac) nothing is mentioned.

**Comment by ghost** [comment]
I also agree that the production tab is a huge feature to this mod. It's even the second video (first video after the overview) in your Colony Manager Youtube Playlist. Granted that was made back in 2016, but still. I definitely feel like this is a feature that should 100% be added back in. Thats really the only reason I downloaded this mod. Been doing a whole "Fluffy Only Mods" playthrough, and its really fun! But it is lacking this core and key feature! :(

**Comment by iamsivil** [comment]
@FluffierThanThou was hoping for an update on if the Production mod you said would be replacing the Production tab from this mod is still happening. (https://github.com/fluffy-mods/ColonyManager/issues/66#issuecomment-304384772). Cheers.
